frontend/components/droplets/lessons/lesson-renderer.tsx

```
@@ -451,61 +450,101 @@ function convertSingleBlock(blockAny: any, blockIndex: number): Block | null {
+        });
+
+        const wrapperClass = isDisplayMode
+          ? "my-4 flex justify-center"
+          : "inline-block";
+
+        return {
+          __component: "droplets.generic",
+          id: blockIndex,
+          content: `<div class="${wrapperClass}">${rendered}</div>`,
+        };
+      } catch (e) {
```

ESLint

[error] 'e' is defined but never used.

(@typescript-eslint/no-unused-vars)